docs/contributors/testing-overview.md

```diff
@@ -429,14 +435,22 @@ DOWNLOAD_THROUGHPUT=125000 npm run test-e2e
 
 Related: https://chromedevtools.github.io/devtools-protocol/tot/Network#method-emulateNetworkConditions
 
-
 ### Core Block Testing
 
 Every core block is required to have at least one set of fixture files for its main save function and one for each deprecation. These fixtures test the parsing and serialization of the block. See [the e2e tests fixtures readme](https://github.com/wordpress/gutenberg/blob/master/packages/e2e-tests/fixtures/blocks/README.md) for more information and instructions.
 
 ## PHP Testing
 
-Tests for PHP use [PHPUnit](https://phpunit.de/) as the testing framework. If you're using the built-in [local environment](/docs/contributors/getting-started.md#local-environment), you can run the PHP tests locally using this command:
+Tests for PHP use [PHPUnit](https://phpunit.de/) as the testing framework. If you're using the built-in [local environment](/docs/contributors/getting-started.md#local-environment), you can run the PHP tests locally using this command.
+
+Note that these commands require the local environment to be initalized first:
+
+```bash
+# Reference the local sourcecode for wp-env in the Gutenberg repo:
+npx wp-env start
```

Comment:
Why `npx` when it can be `npm run`?

Reply:
Npx is shorter and used in other places in the docs, but either works. Is npx included with npm by default? I forget if you have to install anything extra.

Reply:
I think newer versions do bundle it.

I thought we avoided it because it's yet another thing to learn.

It's also confusing for beginners because it won't fail if used somewhere where the dependency is not locally installed, it will just install it and use it globally.

Reply:
cool. I think we may need to update a few more places to remove it, but I can do that here.